### PR TITLE
fix: Booking confirmation: Cancelled event icon regression 

### LINKED ISSUE #14016: [CAL-3274] Booking confirmation: Cancelled event icon regression
Been bugging me for awhile.

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-3274](https://linear.app/calcom/issue/CAL-3274/booking-confirmation-cancelled-event-icon-regression)</sub>

### PR DESCRIPTION
**Pull request by ASQUAREEE**
Fixes: #14016

~I just converted an X icon to an Xcircle icon and fixed the issue.~

### CODE REVIEW

**Review by keithwillcode — DISMISSED** [review]

**Review by PeerRich — DISMISSED** [review]
thats not the right icon. i think we have a svg icon somewhere

File: apps/web/modules/bookings/views/bookings-single-view.tsx [hunk 1]
```diff
@@ -355,7 +355,9 @@ export default function Success(props: PageProps) {
                     <Check className="h-5 w-5 text-green-600 dark:text-green-400" />
                   )}
                   {needsConfirmation && !isCancelled && <Calendar className="text-emphasis h-5 w-5" />}
-                  {isCancelled && <X className="h-5 w-5 text-red-600 dark:text-red-200" />}
+                  {isCancelled && (
+                    <img src="/booking-cancelled.svg" alt="cancelled image" className="h-8 w-8" />
+                  )}
```
Comment (Udit-takkar) [inline apps/web/modules/bookings/views/bookings-single-view.tsx:360]: Doesn't look good in dark mode

File: apps/web/styles/globals.css [hunk 2]
```diff
@@ -17,7 +17,7 @@
   --cal-bg-info: hsla(218,83%,98%,1);
   --cal-bg-success: hsla(134,76%,94%,1);
   --cal-bg-attention: hsla(37, 86%, 92%, 1);
-  --cal-bg-error: hsla(3,66,93,1);
+  --cal-bg-error: hsla(3,66%,93%,1);
```
Comment (Udit-takkar) [inline apps/web/styles/globals.css:20]: This was the bug in light mode. I have removed the svg that you added

**Review by Udit-takkar — APPROVED** [review]
LGTM. I have fixed it now

Screenshot 2024-03-08 at 11 11 43 AM

**Review by PeerRich — APPROVED** [review]

**Pull request merged**